### ISSUE TITLE
508 Accessibility: Color contrast - disabled navigation links

### ISSUE THREAD

**Issue by grossb1** [department-of-veterans-affairs/va.gov-team#21850]
When reviewing the new Project Cheetah workflow for accessibility issues, one the appointments page the 'Choose an appointment type' navigation link failed contrast test when neither of the associated radio buttons were selected.  This issue was also found on the select date page for both the previous and next navigation buttons when they are disabled

.../health-care/schedule-view-va-appointments/appointments/
.../health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/select-date-1

Steps to reproduce:
Navigate to the above pages
Activate your colorblind testing tool (I used the Colorblinding chrome plugin)
Sample the hex color of the disabled navigation link (You can use something like Pixie to do this)
Sample the hex color of the background, it should be #FFFFFF
Go to a website like https://whocanuse.com/ and enter the hex values sampled above
Notice that the contrast test fails.

@outerpress

**Comment by 1Copenut** [comment]
Please see https://github.com/department-of-veterans-affairs/va.gov-team/issues/21848#issuecomment-805969640 for this issue.
